File: day2/go/src/main.go
package main

import (
	"fmt"
)

func main() {
	val,n,v := Day2("../input")
	if n == -1 || v == -1 {
		panic("couldn't find noun verb")
	}
	fmt.Println("Value at position 0", val)
	fmt.Println("noun verb:",n, v)
}

File: day2/rust/src/main.rs
use anyhow::Result;

extern crate day2_lib;
use day2_lib::*;

fn main() -> Result<()> {
    let (rc,noun,verb) = day2("../input")?;
    
    println!("result {}", rc);
    println!("noun verb is {:02}{:02}", noun, verb);

    Ok(())
}